pynamodb/expressions/condition.py

```diff
@@ -14,10 +14,6 @@ def __init__(self, operator, *values):
         self.operator = operator
         self.values = values
 
-    # http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Query.html#DDB-Query-request-KeyConditionExpression
-    def is_valid_range_key_condition(self, path):
```

Comment:
TIOLI: i think this change makes the code less readable

Reply:
I don't think `Condition` needs a `is_valid_range_key_condition` method. I don't think it's a very likely scenario to try and determine it dynamically.

I agree it can be refactored into a method in `TableConnection.query` which is the only place where it's used now.